dockerfiles/theia/Dockerfile

```diff
@@ -92,7 +92,7 @@ ADD src/builder/setup-nodejs.sh /tmp/builder/scripts/setup-nodejs.sh
 # Install supervisor for running theia
 # Install which tool in order to search git
 RUN yum install -y centos-release-scl
-RUN yum install -y rh-git29 scl-utils sudo java-1.8.0-openjdk-devel bzip2 which && \
+RUN yum install -y rh-git29 scl-utils sudo java-1.8.0-openjdk-devel bzip2 which make gcc-c++ && \
```

Comment:
the theia-ide image is only a runtime image so make, gcc and c++ should not be there